content/pages/about/technology-overview.md

```diff
@@ -243,41 +243,41 @@ Servers The following XMPP servers include built-in support for BOSH:
 
 - [ejabberd](http://www.ejabberd.im/)
 - [Jabber XCP](http://www.jabber.com/)
-- [M-Link](http://www.isode.com/products/m-link.html)
+- [M-Link](https://www.isode.com/products/m-link.html)
 - [MongooseIM](https://github.com/esl/MongooseIM)
-- [Openfire](http://www.igniterealtime.org/projects/openfire/index.jsp)
-- [Prosody](http://prosody.im/)
+- [Openfire](https://www.igniterealtime.org/projects/openfire/index.jsp)
+- [Prosody](https://prosody.im/)
 - [Tigase](http://www.tigase.org/)
 
 #### Connection Managers
 
 The following standalone XMPP connection managers can be used with a wide variety of XMPP servers:
 
 - [JabberHTTPBind](http://blog.jwchat.org/jhb/)
-- [Punjab](http://code.stanziq.com/punjab)
+- [Punjab](https://code.stanziq.com/punjab)
 - [node-xmpp-bosh](https://github.com/dhruvbird/node-xmpp-bosh)
 - [rhb](http://rubyforge.org/projects/rhb/)
 
 #### Clients
 
-- [Adium](http://adium.im/)
-- [Gajim](http://gajim.org/)
+- [Adium](https://adium.im/)
+- [Gajim](https://gajim.org/)
 - [JWChat](http://blog.jwchat.org/jwchat/)
-- [Pidgin](http://pidgin.im/)
+- [Pidgin](https://pidgin.im/)
 - [Soashable](http://www.soashable.com/)
-- [SparkWeb](http://www.igniterealtime.org/projects/sparkweb/)
-- [Swift](http://swift.im/swift/)
+- [SparkWeb](https://www.igniterealtime.org/projects/sparkweb/)
+- [Swift](https://swift.im/swift/)
 - [Tigase Messenger](http://www.tigase.org/project/messenger)
 - [Tigase Minichat](http://www.tigase.org/project/minichat)
 
 #### Libraries
 
-- [emite](http://code.google.com/p/emite/) (gwt)
-- [gloox](http://camaya.net/gloox) (C++)
+- [emite](https://code.google.com/p/emite/) (gwt)
```

Comment:
This link has moved to https://github.com/EmiteGWT/emite (it also appears unmaintained, maybe it should just be removed)